File: C++/Pattern1.cpp
// Pattern 1 
// * * * *
// * * * *
// * * * *
// * * * *

#include <iostream>
using namespace std;
void Pattern1(int N){
    //Outer loop to handle rows
    for (int i = 0; i < N; i++){
        // Innet loop to handle colums
        for(int j = 0; j < N; j++){
            cout << "* ";
        }
        cout << endl;
    }

}


int main(){ 
    int N = 5;
    Pattern1(N);
    return 0;
}

File: Java/Pattern1.java
package Java;
// Pattern 1 
// * * * * * * 
// * * * * * * 
// * * * * * * 
// * * * * * * 
// * * * * * * 
// * * * * * * 

class Solution {
    public void pattern1(int N){
        for (int i=0; i < N; i++){
            for(int j = 0; j < N; j++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args){
        Solution pat = new Solution();
        int N = 6;
        pat.pattern1(N);
    }
    
}
